Lib/test/libregrtest/main.py

```diff
@@ -318,9 +318,9 @@ def run_tests_sequentially(self, runtests):
             result = self.run_test(test_name, runtests, tracer)
 
             # Unload the newly imported modules (best effort finalization)
-            for module in sys.modules.keys():
+            for module in list(sys.modules):
```

Comment:
```suggestion
            for module in sys.modules:
```

Reply:
No it does not work, because the dict is modified during iteration.